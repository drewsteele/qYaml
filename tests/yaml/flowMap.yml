# last one should have a null value
{ Boston Red Sox: 1, Detroit Tigers: 2, New York Yankees }
