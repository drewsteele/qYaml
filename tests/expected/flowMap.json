{
  "Boston Red Sox": 1,
  "Detroit Tigers": 2,
  "New York Yankees": null
}
